src/monocle_apptrace/metamodel/inference_base.py

```diff
@@ -0,0 +1,18 @@
+from monocle_apptrace.metamodel.llm import LLM
+from monocle_apptrace.metamodel.provider import ProviderAttributes
+class Inference:
+    def __init__(self, instance):
+        self.instance = instance
+
+
```

Comment:
avoid consecutive new lines, single new line is generally enough
also have new line after all imports, and before beginning of classes and functions

please check other files as well for this consistency